examples/C/H5D/h5ex_d_chunk.c

```diff
@@ -148,9 +148,11 @@ main (void)
         case H5D_CHUNKED:
             printf ("H5D_CHUNKED\n");
             break;
+#if H5_VERSION_GE(1,10,0)
```

Comment:
But what if you have a 1.12 library with a restriction to use only 1.8 APIs (-DUSE_18_API) - is this no longer supported?

Reply:
No, not 1.12 compiled with 1.8 mappings, rather the use of the "USE_1x_API" compile option.

Not expecting it to be a test suite, but it gets included with binaries and the user should be able to test the examples to verify that the library was installed properly.

Reply:
They should use the HDF5 test to make sure hdf5 was installed correctly.  I don't want to add defines to the source code that are specific to hdf5-example.

Reply:
But they are not specific to examples, it is a feature of hdf5.